src/model/ts/default-types.ts

```diff
@@ -1,39 +1,42 @@
 
 /**
- * <p>Alias for Swagger type <a href="https://swagger.io/docs/specification/data-models/data-types/#numbers">number @type integer</a>.</p>
+ * <a href="https://swagger.io/docs/specification/data-models/data-types/#numbers">Number @type integer</a>
  */
-export type Int=number // @asType integer
+export type Int=number // @type integer
 
 /**
- * <p>The date-time notation as defined by <a href="https://tools.ietf.org/html/rfc3339#section-5.6">RFC 3339, section 5.6</a>,
- * for example <code>2014-05-22T10:42:42.542+00:00</code>.</p>
- *
- * <p>See: <a href="https://swagger.io/docs/specification/data-models/data-types/#string">string @format date-time</a>.</p>
+ * <a href="https://swagger.io/docs/specification/data-models/data-types/#numbers">Number @type integer</a> as signed 64-bit long integer.
+ */
+export type Long=number // @type integer @minimum -9223372036854775000 @maximum 9223372036854775000 @format long
```

Comment:
I cannot find any `@format long` on linked page. Only useful thing I can see there is `@format int64`. Why do we have a difference here?

Reply:
This 'long' was needed for NJsonSchema, the .NET code generator.
Maybe int64 works as well and we could replace it to be more compliant.